### ISSUE TITLE
Faculty Bio Requests

### ISSUE THREAD

**Issue by annmarieven** [betheluniversity/cascade#663]
I met with @parlau and here are some questions/requests we have.

- [x] Descriptions
- [x] Department name in lists

**Descriptions**
These used to automatically be created based on this template, "Meet [NAME], [TITLE], at Bethel University." But they're no longer working with the new job title fields. Can we get this working again with the new job title fields? Lauren would like all job titles to be entered into the description automatically (ex: Meet Tom Jones, Department Chair, Associate Professor, and Instructor, at Bethel University) 

**Department name in lists**
Take a look at this one for an example: https://www.bethel.edu/undergrad/academics/biblical-theological/faculty
Professors in CAS always have an area that they teach in within their department. So for example, Christian Collins Winn is "Professor of Historical and Systematic Theology in Biblical & Theological Studies"
We think it's redundant to have the department listed on every bio in the faculty list especially because people are ON the bts faculty list page. So it's assumed these are all BTS faculty.
Can we remove the department from the faculty list but keep it on the full bio pages?
As of now, this is ONLY for CAS faculty.

Let me know if there are any questions!

**Comment by c-schwarze** [comment]
**Descriptions**
This should be completed. Every submitted bio will reflect this change (not any that haven't been submitted in the last few days). I talked with Tim earlier this week and if a faculty member is only in Bethel Seminary, it will have ". . . at Bethel Seminary" instead of ". . . at Bethel University". If you would like, I can take the time to resubmit every bio.

**Department name in lists**
I published all of our faculty listing pages. This should be live.
